Makefile.prow

```diff
@@ -1,4 +1,27 @@
+###############################################################################
+# Licensed Materials - Property of IBM
+# (c) Copyright IBM Corporation 2017, 2019. All Rights Reserved.
+# US Government Users Restricted Rights - Use, duplication or disclosure 
+# restricted by GSA ADP Schedule Contract with IBM Corp.
+#
+# Contributors:
+#  IBM Corporation - initial API and implementation
+###############################################################################
+# Copyright (c) 2020 Red Hat, Inc.
+# Copyright Contributors to the Open Cluster Management project
```

Comment:
After discussing with Ray, I think this line is the only change we should keep in `Makefile.prow`. (We don't need to add IBM and Red Hat copyrights to new files.)

Reply:
k so just `Copyright Contributors to the Open Cluster Management project`?